Expensify/App
# Fix footer position on the login page in mobile View

### Details


### Fixed Issues

$ https://github.com/Expensify/App/issues/6175
$ https://github.com/Expensify/App/issues/6202

### Tests | QA Steps
1. Open Login Page. The footer should be bottom aligned.
2. Focus on the email field, continue button and email input should be visible completely.
3. Click continue,
4. click to enter the password. Password input and continue button and go backlink should be over the keyboard.

Note: **Wide Screen layout is fixed in separate [issue](https://github.com/Expensify/App/issues/5159) and which affect web | desktop | Ipad | Tablet | landscape view and the PR for the same is https://github.com/Expensify/App/pull/6137**.

### Tested On


### Screenshots


#### Web

![image](https://user-images.githubusercontent.com/24370807/140647269-4b2136f5-ee46-4d6f-9d64-c5339e22e3fb.png)

#### Mobile Web

![image](https://user-images.githubusercontent.com/24370807/140647263-1bfcc692-d48c-4dc2-b7c1-c6389ac34c2d.png)

#### Desktop

![image](https://user-images.githubusercontent.com/24370807/140647270-9b3cd9ea-346b-442d-9568-d190c7512433.png)

#### iOS

![Screenshot 2021-11-08 01:13:25](https://user-images.githubusercontent.com/24370807/140659515-f1e46e52-ceee-4bdf-870d-b02b99ea6ed6.png)

#### Android

![Screenshot 2021-11-07 19:05:24](https://user-images.githubusercontent.com/24370807/140647294-00e7df42-d291-4caf-9830-336b0bcba920.png)
![Screenshot 2021-11-07 19:05:06](https://user-images.githubusercontent.com/24370807/140647295-e0e317a2-37bf-4839-8771-1dc7804e358b.png)
![Screenshot 2021-11-07 22:45:58](https://user-images.githubusercontent.com/24370807/140654845-99bad849-5c75-4aa2-a882-270ccdd7e761.png)
